src/components/WithValidation/WithValidation.test.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import { waitFor } from '@testing-library/react';
+
+import { renderWithContext } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+
+import '../../../test/jest/__mock__';
+
+import { WithValidation } from './WithValidation';
+
+const TestComponent = props => {
+  let data = '';
+  const handleChange = () => {
+    const value = props.validate();
+
+    data = value;
```

Comment:
It is no use to declare **value** variable

Reply:
Removed that variable.